src/components/containers/TraceAccordion.js

```diff
@@ -27,9 +29,13 @@ export default class TraceAccordion extends Component {
     const {canAdd, children} = this.props;
 
     const addButton = canAdd && (
-      <button className="panel__add-button" onClick={this.addTrace}>
-        + Trace
-      </button>
+      <Button
+        className="js-add-trace-button"
+        variant="primary"
+        onClick={this.addTrace}
+        icon={<PlusIcon />}
```

Comment:
or PlusIcon could be the default in the Button component?

Reply:
or maybe not, if we're including modals to the editor (which I'm not sure we will be), maybe we want more flexibility on that button

Reply:
Maybe... I think it might be better to leave no default icon, because there are buttons that don't have icons.